xchange-binance/src/main/java/org/knowm/xchange/binance/service/BinanceAccountService.java

```diff
@@ -91,25 +94,19 @@ public AccountInfo getAccountInfo() throws IOException {
     try {
       List<Wallet> wallets = new ArrayList<>();
       List<OpenPosition> openPositions = new ArrayList<>();
-
-      if (exchange.usingSandbox()) {
-        if (exchange.isFuturesSandbox()) {
-          BinanceFutureAccountInformation futureAccountInformation = futuresAccount();
-          wallets.add(BinanceAdapters.adaptBinanceFutureWallet(futureAccountInformation));
-          openPositions.addAll(
-              BinanceAdapters.adaptOpenPositions(futureAccountInformation.getPositions()));
-
-        } else {
+      switch (exchange.getExchangeSpecification().getExchangeSpecificParametersItem(
+          EXCHANGE_TYPE).toString()) {
+        case "SPOT": {
```

Comment:
please use the enum instead of its string representation

Reply:
done